cmd/shutdown_windows.go

```diff
@@ -14,29 +14,12 @@ limitations under the License.
 package cmd
 
 import (
-	"fmt"
 	"os"
 	"os/signal"
 	"syscall"
-
-	"github.com/dapr/cli/pkg/print"
-	"golang.org/x/sys/windows"
 )
 
 func setupShutdownNotify(sigCh chan os.Signal) {
 	// This will catch Ctrl-C.
 	signal.Notify(sigCh, syscall.SIGTERM, syscall.SIGINT)
-
```

Comment:
Why is this being removed?

Reply:
As mentioned in the description, this is no longer needed as `pkg/standalone/stop_windows.go` is not using named events. Instead, it's using TaskKill.

Reply:
If this is only going to be the signal Notify call alone, should the shutdown.go file be used for all OSes ?

Reply:
Yes, missed that. Let me do that.